src/components/FlowRunFilteredList.vue

```diff
@@ -57,7 +58,7 @@
     emit('update:states', states.value)
   }
 
-  const sort = ref<FlowRunSortValues>('START_TIME_DESC')
+  const sort = ref<FlowRunSortValues>(props.sort ?? 'START_TIME_DESC')
```

Comment:
Rather than adding an additional prop. We should just grab the sort from the filter since sort is already part of that type. Otherwise it could get confusing if you have a sort on your filter and it gets ignored.

Reply:
Yeah good point.  I think your point about potentially splitting these out into a new component makes sense!